src/main/java/io/excaliburfrc/robot/subsystems/Intake.java

```diff
@@ -154,6 +164,10 @@ public void close() {
     static final double upperShootDutyCycle = 0.6;
   }
 
+  private double getDistanceToTarget(PhotonTrackedTarget target) {
+    return PhotonUtils.calculateDistanceToTargetMeters(CAMERA_HEIGHT, TARGET_HEIGHT, CAMERA_PITCH, Units.degreesToRadians(target.getPitch()));
+  }
```

Comment:
where is this going to be used?

Reply:
to create a trajectory to the ball

Reply:
can you add #33 to this?

Reply:
doesn't it include https://github.com/ExcaliburFRC/RapidReact2022/issues/38 also?

Reply:
actually, I forgot that both need #36